app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/view/fragment/ListFragment.kt

```diff
@@ -21,16 +20,21 @@ import kotlinx.android.synthetic.main.list_fragment.view.*
 import javax.inject.Inject
 
 class ListFragment : Fragment() {
```

Comment:
Name of fragment is generic ? I see below it is for Movies. Maybe change name.

Reply:
Yes - that's true. I will change it to "MovieListFragment"